.github/workflows/release-asset.yml

```diff
@@ -0,0 +1,57 @@
+name: Elixir CI
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    name: Build and publish release asset
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Elixir
+        uses: actions/setup-elixir@v1
+        with:
+          elixir-version: '1.10.4'
+          otp-version: '23.1'
+
+      - name: Restore dependencies cache
+        uses: actions/cache@v2
+        with:
+          path: deps
+          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
+          restore-keys: ${{ runner.os }}-mix-
+
+      - name: Install dependencies
+        run: mix deps.get
+
+      - name: Build release
+        run: |
+          mkdir ./release
+          mix compile
+          mix elixir_ls.release -o ./release
```

Comment:
Do we need to add `MIX_ENV=prod`?

Reply:
We haven't been using that up until now (you can see the instructions we do use at https://github.com/elixir-lsp/elixir-ls/blob/3a05edf1a0fc36fdfeb136f8856746edec1adfc8/DEVELOPMENT.md#packaging) but in the future it would be good to try with `MIX_ENV=prod`

Reply:
Should I add it now or leave it as is?